recipes/libm2k/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "libm2k" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 64f62cf9a72d625c4a4344a1984a2f8c5d4a95aeaed0026a0cba7ac0c0ca7390
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libm2k', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - {{ cdt('libudev-devel') }}  # [linux]
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - libiio
+    - libusb >=1
```

Comment:
The build output seems to indicate that `libusb` is not a necessary dependency (directly, but `libiio` will still install it):
```
WARNING (libm2k): run-exports library package conda-forge::libusb-1.0.23-he1b5a44_3 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```